.pipelines/templates/PackageTestResultsAnalysis.yml

```
@@ -43,6 +43,7 @@ steps:
         displayName: "Authenticate to custom pip artifact feeds"
 
   - bash: pip3 install junit_xml
+    retryCountOnTaskFailure: 1
```

Eh, let's make it 3 for good measure. Just so we if we ever hit this issue again we can more confidently say it's persistent.
```suggestion
    retryCountOnTaskFailure: 3
```